doc/faq.rst

```diff
@@ -470,6 +470,15 @@ documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-a
 
 .. _faq_mkl_threading:
 
+Why do estimators sometimes produce unexpected/spurious results?
```

Comment:
I would frame this as float32 vs float64 rather than unexpected/spuriours results which seems way too vague